web/app_dev.php

```diff
@@ -13,16 +13,14 @@
 /*
 if (isset($_SERVER['HTTP_CLIENT_IP'])
     || isset($_SERVER['HTTP_X_FORWARDED_FOR'])
-    || !(in_array(@$_SERVER['REMOTE_ADDR'], ['127.0.0.1', 'fe80::1', '::1']) || php_sapi_name() === 'cli-server')
```

Comment:
This code block is entirely commented :)

https://github.com/Elao/symfony-standard/blob/master/web/app_dev.php#L13